### PR TITLE
Query update table

### PR DESCRIPTION
**Pull request by xtrahuman**
# Query and update table
- created a new column called species
- inserted more data to data.sql
- added more queries to query.sql

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -7,4 +7,133 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' or name = 'Pikachu';
...
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
+
+-- update the animals table by setting the species column to unspecified
+UPDATE animals
+SET species = 'unspecified';
+
+-- Rollback database
```
Comment (SajjadAhmad14) [inline queries.sql:17]: Please add queries before and after rollback to verify explicitly that the change was made.

File: queries.sql [hunk 2]
```diff
@@ -7,4 +7,133 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' or name = 'Pikachu';
...
+WHERE name LIKE '%mon';
+
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species != 'digimon';
+
+-- Commit the transaction.
+
```
Comment (SajjadAhmad14) [inline queries.sql:38]: Please add query before commit as well to verify explicitly that change was made.

File: queries.sql [hunk 3]
```diff
@@ -7,4 +7,133 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' or name = 'Pikachu';
...
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species != 'digimon';
```
Comment (SajjadAhmad14) [inline queries.sql:35]: Please update the animals table by setting the species column to `digimon` for all animals that have a name ending on `mon`. Currently, you are not checking for ending name.
Please check for all animals that don't have species already set and then set the species column to pokemon.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Tochukwu, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (xtrahuman) [reply queries.sql:17]: resolved

File: queries.sql (hunk 2 above)
Comment (xtrahuman) [reply queries.sql:38]: resolved

File: queries.sql (hunk 3 above)
Comment (xtrahuman) [reply queries.sql:35]: it was stated in:
Line 27
`UPDATE animals
SET species = 'digimon'
WHERE name LIKE '%mon';`

and Line 33
`UPDATE animals
SET species = 'pokemon'
WHERE species != 'digimon';
`

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @xtrahuman 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**